css/styles.css

```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap");
+
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  html,
+  body {
+    height: 100%;
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-content: center;
+    background: #efefef;
+    margin: 0 auto;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+  }
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+
+  a:link {
+    text-decoration: none;
+  }
+
+  .header {
+    position: relative;
+    width: 100%;
+    height: 48px;
+    margin: 0 auto;
+    background-color: #fff;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    gap: 45%;
+  }
+
+  .logo {
+    position: relative;
+    width: auto;
+    height: auto;
+    padding: 0 5%;
+    font-family: "Poppins", sans-serif;
+    color: #0f27ff;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 20px;
+    order: 0;
+  }
+
+  .nav-bar {
+    position: relative;
+    padding: 0 5%;
+    width: auto;
+    margin: 0;
+    height: auto;
+    order: 1;
+  }
+
+  .mobile-menu-icon-wrapper {
+    position: relative;
+    width: auto;
+    height: auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .mobile-menu-icon {
+    position: relative;
+    width: 24px;
+    height: 24px;
+    padding: 20%;
+  }
+
+  .headline-section {
+    position: relative;
+    height: 100vh;
+    width: 100%;
+    margin: 0 auto;
+    border-bottom-left-radius: 5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    justify-content: center;
+    gap: 1.2rem;
+    background-color: #fff;
+    background-image: url("../images/bgd/mob-header-bg.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    overflow: hidden;
+  }
```

Comment:
- **[OPTIONAL]** Kindly add space at the botton to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/227408252-a4aff145-3db6-4efa-827e-8e99ebd7a24a.png) | ![image](https://user-images.githubusercontent.com/58642949/227408320-fbfe36c3-db62-4689-a0b4-48cb8880d288.png) |